auth-web/devops/cloudbuild-pr.yaml

```diff
@@ -21,7 +21,9 @@ steps:
     env:
       - 'VUE_APP_ADDRESS_COMPLETE_KEY=$_VUE_APP_ADDRESS_COMPLETE_KEY'
       - 'VUE_APP_AUTH_LD_CLIENT_ID=$_VUE_APP_AUTH_LD_CLIENT_ID'
+      - 'VUE_APP_CORP_FORMS_URL=$_VUE_APP_CORP_FORMS_URL'
       - 'VUE_APP_REGISTRIES_SEARCH_API_KEY=$_VUE_APP_REGISTRIES_SEARCH_API_KEY'
+      - 'VUE_APP_SOCIETIES_URL=$_VUE_APP_SOCIETIES_URL'
```

Comment:
If the environment variables is not the secret or password, don't need to set it in this file.

Reply:
As with the Corp Forms URL above, we can get this from 1Password, so shouldn't it be here?

Could I request from you a demo of how env keys are populated now, maybe at the Developer Sprint Demo meeting today? I think I don't understand this very well :)